src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,26 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
+  }
+
   public String getBackgroundColor() {
     return component.get(0).getCssValue("background-color");
   }
 
+  public String getBorderStyle() {
+    JavascriptExecutor js = (JavascriptExecutor)driver;
+    WebElement selectedComponent = component.get(0);
+    wait.forElementVisible(selectedComponent);
+    selectedComponent.click();
+
+    String script = "return window.getComputedStyle("
```

Comment:
if you will use `execute` method, you should remove `return` it is doing this already